beaglebone/udc-bsmp.txt

```diff
@@ -102,12 +102,11 @@ IA-20RaPS02:PS-UDC-AS       IA-20RaCtrl:PS-DCLink-AS2 2
 # ______ TS ______
 
 # --- TS Dipole ---
-LA-RaPS02:PS-UDC-TS1        TS-Fam:PS-B 1
-LA-RaPS02:PS-UDC-TS2        TS-Fam:PS-B-Slave 2
+LA-RaPS02:PS-UDC-TS         TS-Fam:PS-B 1
```

Comment:
@xresende, I think there's a 1 missing at the end of the UDC definition:

LA-RaPS02:PS-UDC-TS1

Reply:
done.thanks!